tornado/locale.py

```diff
@@ -248,6 +248,10 @@ def get_closest(cls, *locale_codes: str) -> "Locale":
                 return cls.get(code)
             if parts[0].lower() in _supported_locales:
                 return cls.get(parts[0].lower())
+            if len(parts) == 1:
```

Comment:
Should this be part of the `elif len(parts)` blocks above?

Reply:
Using the `elif len(parts)` would place it higher in the order of operations.  There are two returns after the conditional statement, so I wanted it to be the last thing it tries after the others fail.